templates/ui-frameworks/lit/entry-type/ui/src/{{dna_role_name}}/{{coordinator_zome_manifest.name}}/{{#if crud.update}}edit-{{kebab_case entry_type.name}}.ts{{¡if}}.hbs

```diff
@@ -12,7 +12,6 @@ import { decode } from '@msgpack/msgpack';
   {{/each}}
 {{/uniq_lines}}
 
-import { sharedStyles } from '../../shared-styles';
```

Comment:
Why are the sharedStyles being removed here? I'm assuming this will affect for example the button appearance?

Reply:
I realized I could avoid this repetition by loading the styles to the `index.html` file

https://github.com/holochain/scaffolding/blob/5fc09c3d3ae34df0fe39508ee3b15fc62970dfd6/templates/ui-frameworks/lit/web-app/ui/index.html.hbs#L12

Reply:
Just reverted the change though, it looks like loading the css from the html doesn't work as I expected

Reply:
Yes I think it's required inside the component itself because lit uses shadow root which ignores global styles.